### ISSUE TITLE
Thumb sucking

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10673]
**Preferred term label:**
Thumb sucking

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0040068

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Paronychia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Do you mean that Thumb sucking is a risk factor for Paronychia? In this case, according to this article, https://www.ncbi.nlm.nih.gov/books/NBK556112/, "thumb sucking continues, negative consequences can occur, such as a deformity of the nail or paronychia", so I wonder if we should create a term such as

History of thumb sucking past the age of four years

or something along those lines?

**Comment by MickeySegal** [comment]
That is fine.  

I also see that we now use this finding also for Herpetic whitlow

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001213
- label: History of thumb sucking past the age of four years
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:32310572
- def: Thumb sucking is a behavior that ceases spontaneously in most children between 2 and 4 years of age. If thumb sucking continues, negative consequences can occur, such as a deformity of the nail or paronychia. This term refers to a history of habitual thumb sucking past the age of 4 years of life.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10673
- synonyms:
